### DISCUSSION TITLE
Mac clang: error: linker command failed with exit code 1

### DISCUSSION THREAD

**Discussion by vanessachung97** [su2code/SU2#2200] [Q&A]
Hi, I was able to get passed the ./configure stage but executing make has been an issue. Could someone please explain how to fix this error:

ld: archive member 'libcgns.a' not a mach-o file in '/Users/vanessachung/Downloads/SU2-7.0.4/Common/lib/libSU2.a'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [../bin/SU2_CFD] Error 1
make: *** [all-recursive] Error 1

Thank you in advance!

**Comment by bigfooted** [comment]
Hi,
We do not use configure/make anymore. Did you follow this manual:
https://su2code.github.io/docs_v7/Build-SU2-Linux-MacOS/#meson-and-ninja

We now use meson and ninja:
 ```
./meson.py build
./ninja -C build install
```

**Reply by vanessachung97** [reply]
Hi, thank you for your reply. I did not see bootstrap, configure, and make in the 8.0.0 release and thought it was a problem on my end, and therefore, attempted to install the 7.0.4 release.

Does that mean for 8.0.0, all I need to do is meson and ninja and add it to my path?

**Reply by vanessachung97** [reply]
To follow up, I tried to execute sudo ./ninja -C build install and this is the error message I'm getting:

clang: error: linker command failed with exit code 1 (use -v to see invocation)
[804/810] Compiling C++ object SU2_DEF...rivers_CDiscAdjDeformationDriver.cpp.o
ninja: build stopped: subcommand failed.

**Reply by bigfooted** [reply]
Maybe you are missing some packages that are installed by default? 
`./meson.py build --reconfigure -Denable-cgns=false -Denable-tecio=false --prefix=/home/vanessachung97/SU2`
You can use --reconfigure to overwrite the previous configuration and you can use --prefix to point to your SU2 folder. ninja will place the executable in the folder /home/vanessachung97/SU2/bin
If it still fails, please post the entire output of what meson prints to the screen

**Reply by vanessachung97** [reply]
performing ./ninja -C build install, then follow by sudo ./ninja -C build install after the error worked for me.
